statsmodels/tsa/api.py

```diff
@@ -24,4 +24,6 @@
 from .statespace.dynamic_factor import DynamicFactor
 from .regime_switching.markov_regression import MarkovRegression
 from .regime_switching.markov_autoregression import MarkovAutoregression
-from .holtwinters import ExponentialSmoothing, SimpleExpSmoothing, Holt
\ No newline at end of file
+from .holtwinters import ExponentialSmoothing, SimpleExpSmoothing, Holt
+
+from .automatic import api as automatic
```

Comment:
If you were to define `__all__` and change the `from .stattools import *` to explicit imports, that would save me some teeth-pulling.